src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -14,6 +14,7 @@
 using System.Diagnostics;
 using System.Threading;
 using tasks = System.Threading.Tasks;
```

Comment:
You can avoid this articial `tasks` alias that you have to use everywhere by adding these two usings instead:

```csharp
using System.Threading.Tasks;
using Task = System.Threading.Tasks.Task; // disambiguate with MS.VS.Shell.Task
```